skops/io/_sklearn.py

```diff
@@ -21,10 +21,10 @@
     SquaredLoss,
 )
 from sklearn.tree._tree import Tree
-from sklearn.utils import Bunch
```

Comment:
this is because we load and create the class of the object whatever it was now in `dict_get_instance` instead of creating a `dict`.